.circleci/config.yml

```diff
@@ -28,6 +35,12 @@ jobs:
       - run:
           name: Check installed dependencies are compatible
           command: |
+            # Run in a fresh virtual environment, to avoid conflicts with preinstalled packages.
+            virtualenv -p python3.6 .venv
+            source .venv/bin/activate
+            pip install --progress-bar=off -U pip
+            # Ensure consistency between tensorflow and tensorflow-probability
+            pip install --progress-bar=off tensorflow==2.6.2 tensorflow-probability==0.14.0
```

Comment:
same point about 2.6 / 0.14 so we would automatically get 2.6.3 / 0.14.1 ...

Reply:
Actually, to keep it consistent across the file, how about specifying tensorflow versions as config constants or even turning this into a [reusable command](https://circleci.com/docs/2.0/reusing-config/) ?

Reply:
> Actually, to keep it consistent across the file, how about specifying tensorflow versions as config constants or even turning this into a reusable command ?

I have refactored this config to use reusable commands.